src/webviews/createComponent.ts

```diff
@@ -21,16 +22,67 @@ interface CreateComponentWebviewContent {
   value: {};
 }
 
-export interface CreateComponent {
-  type: "createComponent";
+interface OpenDialogResponse {
+  type: "openDialogResponse";
   value: {
-    componentName: string;
-    version: string;
-    provider: string;
-    scheme: string;
+    fileUri: string;
+    forResourceId: number;
   };
 }
 
+export interface FileResource {
```

Comment:
We should use the ocm types already defined here: https://github.com/open-component-model/vscode-ocm-tools/blob/main/src/ocm/ocmv3.d.ts